src/main/java/legend/game/combat/SEffe.java

```diff
@@ -222,9 +227,11 @@ private SEffe() { }
 
   private static final Value _800fb7bc = MEMORY.ref(1, 0x800fb7bcL);
 
+  /** Some kind of mysterious global 2-hit addition array */
   private static final Value _800fb7c0 = MEMORY.ref(1, 0x800fb7c0L);
 
-  private static final Value _800fb7f0 = MEMORY.ref(1, 0x800fb7f0L);
+  /** Fours sets of color values used for addition overlay borders; only last actually used */
+  public static final ArrayRef<UnsignedByteRef> additionBorderColors_800fb7f0 = MEMORY.ref(1, 0x800fb7f0L, ArrayRef.of(UnsignedByteRef.class, 0xc, 0x1, UnsignedByteRef::new));
```

Comment:
```suggestion
  public static final ArrayRef<UnsignedByteRef> additionBorderColours_800fb7f0 = MEMORY.ref(1, 0x800fb7f0L, ArrayRef.of(UnsignedByteRef.class, 0xc, 0x1, UnsignedByteRef::new));
```
We use the _proper_ spelling everywhere else

Reply:
Don't tempt me to change *all* of the other spellings to be consistently correct. :stuck_out_tongue: